airflow/www_rbac/views.py

```diff
@@ -1279,7 +1279,7 @@ def landing_times(self, session=None):
             base_date = dag.latest_execution_date or timezone.utcnow()
 
         dates = dag.date_range(base_date, num=-abs(num_runs))
-        min_date = dates[0] if dates else datetime(2000, 1, 1)
+        min_date = dates[0] if dates else datetime(2000, 1, 1, tzinfo=timezone.utc)
```

Comment:
Idem

Reply:
Separate PR in https://github.com/apache/incubator-airflow/pull/3303